build/publish.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# run npm publish for every tag; run as part of CI on AWS CodeBuild
+git tag --points-at HEAD | while read tag; do
+
+    disttag=$(echo $tag | grep -oP '^(v|style-spec@)\d+\.\d+\.\d+-?\K(\w+)?')
+    disttag=${disttag:+next}
+    disttag=${disttag:-latest}
```

Comment:
Can we require an explicit tag to ensure that beta releases aren't accidentally published as latest? If I recall correctly a similar change was made to mbx-cli after I made that mistake, it seems maybe even easier to make here since there's no dry-run.

Reply:
>My best understanding of what's happening here is that if the tag is set to any value, it becomes "next." A null tag becomes "latest."

I was confused by this too. The tag is piped through `grep` which I believe is looking to match any word characters after `{v/style-spec@}digit.digit.digit-` so that if the tag contains `-alpha` or `-beta`, `disttag` becomes `alpha` or `beta`. If the tag is say `v.2.8.0`, `disttag` is null. Then it uses [shell parameter expansion](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) to substitute the appropriate tag.

`tag="v2.8.0"`
- disttag is null
- disttag:+next -> null
- disttag:-latest -> "latest"

`tag="v2.8.0-beta.1"`
- disttag is "beta"
- disttag:+next ->"next"
- disttag:-latest -> "next"

Reply:
Thanks for your working testing this and the explanation Ryan! I was confused about where the `disttag` value was coming from as our documentation caught up with this change.  This looks great to me.